src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+
+The Tanzu Design for VMware Edge Compute Stack described here offers a best practice for running Tanzu at edge locations where the VMware Edge Compute Stack is already deployed. We’ll specifically address these challenges at the edge:
+
+- Running and managing Kubernetes infrastructure
+- Providing a consistent development and deployment environment for applications across the cloud, datacenter, and edge
+- Managing a fleet of Kubernetes clusters
+- Proactively monitoring clusters and apps
+- Ensuring security and compliance of edge environments.
+
+On top of the VMware Edge Compute Stack, the VMware Tanzu Edge Solution Architecture comprises core VMware Tanzu capabilities, including:
+
+- Unified Kubernetes Runtime: VMware Tanzu Kubernetes Grid (TKG) provides a consistent, upstream-compatible implementation of Kubernetes, that is tested, signed, and supported by VMware.
+- Global Multi-cluster Management: VMware Tanzu Mission Control (TMC) is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds, regardless of where they reside.
+- Full-stack Observability: VMware Tanzu Observability by Wavefront (TO) is a high-performance streaming analytics platform that supports 3D observability (metrics, histograms, traces/spans). You can collect data from many services and sources across your entire application stack, including edge sites.
+
+This solution architecture is intended for anyone responsible for designing, implementing, managing and securing edge locations, including DevOps admins, VI admins, platform operators, and security teams.
+
+## Topology
+
+There are multiple topologies that can be leveraged for edge deployments. A centralized model, edge site are managed from a central hub (Hub and Spoke topology). A decentralized model is also available where each edge site is managed locally.
+This solution guide is focused on the centralized model where a vCenter is deployed at the main datacenter location and a datacenter with vSphere cluster at each edge location. It is assumed that the vSphere infrastructure is already deployed.
+An Edge location can be a storefront (point of sale), manufacturing facility, medical office, or other IT infrastructure located in a remote, distributed site. Although these locations are important to your organization, they usually run up to 100 tenant virtual machines. Because of the critical nature of the workloads, dependencies on a Wide Area Network (WAN) connection must be limited.
+
+### Hub and Spoke
+
+In this topology, the management plane is entirely residing in the main datacenter. The centralized management plane composed of vCenter, NSX Advanced Load Balancer controller and TKG Management cluster are all in the same central location.
+Only the TKG Workload clusters are created at the edge along with Load Balancing components (Service Engines). This design provides a minimal footprint at the edge.
```

Comment:
Only the Tanzu Kubernetes Grid workload clusters and the NSX Advanced Load Balancer Service Engines (SEs) are created at the edge.

Reply:
Global: Use expanded product names - Tanzu Kubernetes Grid, Tanzu Mission Control, NSX Advanced Load Balancer, Tanzu Observability, Tanzu Service Mesh. 
Exception - Tables and figures. TKG, TMC, NSX ALB, TO, TSM.